src/CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ if(SECP256K1_INSTALL)
     NO_SET_AND_CHECK_MACRO
   )
   write_basic_package_version_file(${PROJECT_NAME}-config-version.cmake
-    COMPATIBILITY SameMajorVersion
+    COMPATIBILITY SameMinorVersion
```

Comment:
note: If we'd follow semver strictly, then that's not true for versions pre 1.0.0. 

But I think it's good to add "SameMinorVersion" here. It seems that we have agreed on bumping minor for incompatible API changes even pre 1.0.0. (We bumped 0.2.0 to 0.3.0 for example because of this.)

(The rationale in semver is that you should release 1.0.0 as soon as people rely on the API. We don't do that either, so I think what we do makes sense in the end, even though we deviate from semver.)